lib/models/meeting.js

```diff
@@ -1,10 +1,10 @@
 const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
 
 const meetingSchema = new mongoose.Schema(
```

Comment:
Since we imported `Schema` separately we could do but that is up to you.
```suggestion
const meetingSchema = new Schema(
```